Controller/CustomItem/ExportController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\CustomObjectsBundle\Controller\CustomItem;
+
+use Mautic\CoreBundle\Controller\AbstractFormController;
+use MauticPlugin\CustomObjectsBundle\CustomItemEvents;
+use MauticPlugin\CustomObjectsBundle\Event\CustomItemExportSchedulerEvent;
+use MauticPlugin\CustomObjectsBundle\Model\CustomItemExportSchedulerModel;
+use MauticPlugin\CustomObjectsBundle\Provider\CustomItemPermissionProvider;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpFoundation\File\Exception\FileNotFoundException;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+class ExportController extends AbstractFormController
+{
+    private CustomItemPermissionProvider $permissionProvider;
+
+    private CustomItemExportSchedulerModel $model;
+
+    public function __construct(
+        CustomItemPermissionProvider $permissionProvider,
+        CustomItemExportSchedulerModel $model
+    ) {
+        $this->permissionProvider = $permissionProvider;
+        $this->model              = $model;
+    }
+
+    /**
+     * @throws \Doctrine\ORM\ORMException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     * @throws \MauticPlugin\CustomObjectsBundle\Exception\ForbiddenException
+     */
+    public function exportAction(int $object): Response
+    {
+        $this->permissionProvider->canCreate($object);
+
+        $customItemExportScheduler = $this->model->saveEntity($object);
+
+        /** @var EventDispatcherInterface $dispatcher */
+        $dispatcher = $this->get('event_dispatcher');
```

Comment:
```suggestion
        $dispatcher = $this->get('event_dispatcher');
        \assert($dispatcher instanceof EventDispatcherInterface);
```

`\assert` is preferred over `@var`